guide/onboarding/funding-a-wallet.md

```diff
@@ -22,101 +30,281 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+Present the payment request to the user with an easily scannable QR code alongside having the option to copy and share it.
 
-Bitcoin ATMs are another great option for new users to exchange cash for bitcoin in person. These machines also let users trade their bitcoin for cash, which can be useful if they are somewhere new or foreign.
+Alongside the payment request, explain to the user that a payment channel will be opened with the received amount and that a minimum amount should be sent. This amount is to cover the network fees for opening a new payment channel.
 
-Finally, new users can buy bitcoin directly from other bitcoin users such as friends, other users at a meetup, or intermediary services like [localbitcoins](https://localbitcoins.com/).
+For users who want to [manually open a channel]({{'/guide/onboarding/funding-a-wallet/#manual-channel-opening' | relative_url}}) this screen should be skippable.
 
 </div>
 
-{% include /tip/open.html label="Tip: Teach users how to transfer" icon="info" color="blue" %}
+{% include /tip/open.html label="⚡ Pay-to-open" icon="info" color="blue" %}
+
+Offering a pay-to-open Lightning service allows users to fund their wallet directly with a Lightning payment.
+
+{% include /tip/close.html %}
+
+### Network sync
+
+Sync methods like compact block filters (BIP157/158) to sync to the Bitcoin network take some time to finish. If your application uses them, you should communicate to the user that their funds will not appear until the sync is complete. You may even consider not presenting a payment request until this sync is complete. This can help prevent users from getting confused about their funds not showing up.
+
+It's recommended to start any required syncing as soon as the user opens your application to limit wait times.
+
+## 3. Sending the bitcoin
+
+The next step for the user is to send bitcoin to the wallet.
+
+Once the bitcoin has been sent, notify the user once it's received. This should be followed by notifying the user that their wallet is being setup. The setup process being the confirmation of the first transaction and opening of a new payment channel.
 
-Users funding new wallets from an exchange could potentially come into contact with unknown concepts, such as a bitcoin address, QR code, or transaction confirmations. This is an excellent opportunity to explain the exchange to personal wallet flow.
+{% include /tip/open.html label="⚡ Zero confirmation channels" icon="info" color="blue" %}
+
+Offering a zero confirmation channel Lightning service allows users to use their wallet without having to wait for confirmations.
 
 {% include /tip/close.html %}
 
-{% include fact/pros.html %}
-  - An easy and popular way to acquire bitcoin
-{% include fact/close.html %}
+For beginner users, it's recommended to open a payment channel for them automatically once the initial transaction is confirmed. This way, they don't have to manually find a peer and open a channel.
```

Comment:
Who are they opening channel to? Is a wallet provider opening it to them, or is user opening to someone automatically. I think we should be clear here on `who is who` so people don't get lost, I did get lost a bit here tbh.

And if a wallet provider is opening a channel on behalf of users, isn't that then a lightning service? Confused since you specifically said this flow excludes any lightning service.

Reply:
To the wallet providers Lighting node automatically yes. They aren't opening a channel for the user the application just automatically opens to a pre-selected node (the wallets ideally). So I wouldn't call this a Lightning service per se. Will clarify this some more

Reply:
So what you're suggesting here is just auto-channel opening to a random node? I think that's really bad practice, but if it opens channels to the provider's node only, that to me is a service, because it won't work unless provider provides certain % on top of the liquidity like Breez does.

Reply:
No I'm not recommending that, it would be to the wallet providers node like Blixt does. 

You can have a channel opened to the wallet providers node without any interaction from them, service, or additional fees. It would only have outbound capacity in this scenario. 

It's just creating and broadcasting a channel open automatically without the user manually doing the process.

Reply:
I see now what you mean, can we then revise this so it's a bit clearer and make sure to tell about the tradeoffs like having only onbound capacity?